mcp/src/main/java/io/modelcontextprotocol/spec/McpServerSession.java

```diff
@@ -53,6 +54,8 @@ public class McpServerSession implements McpSession {
 
 	private final AtomicInteger state = new AtomicInteger(STATE_UNINITIALIZED);
 
+	private final AtomicReference<LoggingLevel> minLoggingLevel = new AtomicReference<>(LoggingLevel.INFO);
```

Comment:
We probably just need a volatile field for an enum that we just set.

Reply:
sure